base/deprecated.jl

```diff
@@ -1599,6 +1599,15 @@ end
 # issue #6466
 # `write` on non-isbits arrays is deprecated in io.jl.
 
+# PR #23035
+# also uncomment constructor tests in test/linalg/tridiag.jl
+function SymTridiagonal(dv::AbstractVector{T}, ev::AbstractVector{S}) where {T,S}
```

Comment:
could also use a deprecation bullet in news (or could put it for both in whichever merges second)